src/routes/bounce_rules_detailed.jsx

```diff
@@ -29,49 +30,52 @@ export default class BounceRuleDetailedPage extends React.Component {
     this.handleCancelConfirmation = this.handleCancelConfirmation.bind(this);
     this.handleSaveConfirmation = this.handleSaveConfirmation.bind(this);
     this.onChangeRuleInt = this.onChangeRuleInt.bind(this);
+    this.updatePageIndex = this.updatePageIndex.bind(this);
   }
 
   async componentDidMount() {
-    const { match, location } = this.props;
+    const { match } = this.props;
+    const { changelogLimit, pagesToDisplay } = this.state;
     getChangelog(match.params.bounceRuleId)
       .then(res => {
         const { data } = res;
         this.setState({
           changelog: data.reverse(),
+          pagesToDisplay:
+            data.length <= changelogLimit * pagesToDisplay
+              ? Math.ceil(data.length / changelogLimit)
+              : 5,
```

Comment:
On the next PR let's refactor this to not be a magic number but some constant variable name to more explicitly tell us what 5 means like how we did in the bounce rules page!